src/main/java/in/micheal/dao/DebtUserDetailsDAO.java

```diff
@@ -18,6 +18,11 @@ private DebtUserDetailsDAO() {
 		// default constructor
 	}
 
+	static final String BOOK = "taken_book";

```

Comment:
Add private to reduce the scope of the variable.

Reply:
ok sir